src/Pricing/ModelPricing.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Outl1ne\NovaOpenAI\Pricing;
+
+class ModelPricing
+{
+    protected string $model;
+
+    public function __construct(protected readonly Pricing $basePricing)
+    {
+        //
+    }
+
+    public function model(string $model): self
+    {
+        $this->model = $model;
+        return $this;
+    }
+
+    public function calculate(int $inputTokens, int $outputTokens)
+    {
+        if (!$this->basePricing->pricing) return null;
+
+        return $this->basePricing->pricing->models->{$this->model}->input * $inputTokens / 1000
```

Comment:
Kui model unconfigured, siis megacrash

Kinda jääb arusaamatuks kuidas ->pricing->models->{$this->model} üldse töötab - kas see töötab?

Reply:
Tuleb crash jep, võiks mingi validatsiooni ja arusaadavate Exceptionite throw'imise teha. Mis sa muidu arvad, kui kasutada mudelit, mille pricing meil pole teada ja seega ei tea, kas on mingi uus mudel, siis kas peaksime pricingus errori throw'ima või silently null tagastama? Ma ise kaldun sinnapoole, et kui pricing või mudeli olemasolu pole teada, siis tagastaks null.